evalytics/server/core.py

```diff
@@ -0,0 +1,17 @@
+
+
```

Comment:
to be honest, I start doubting about this "select which implementation" castle we're building here. I start thinking that it's not needed actually. 

#trolling If we change some names adding some fancy words like `Provider` `ManagerProvider` ... we could turn this into a Java app